.travis/setup_mo.sh

```diff
@@ -6,45 +6,55 @@ if [[ -z $TRAVIS_BUILD_DIR ]]; then
     export TRAVIS_BUILD_DIR=`pwd`;
 fi
 
+# Replace the default client certificate with the new one to make sure mo keeps working
+cp ${TRAVIS_BUILD_DIR}/mongo-orchestration/ssl/client.pem ${MO_PATH}/lib/client.pem
```

Comment:
> Note that due to us using an old stable version of mongo-orchestration (which has since not been replaced), we need to overwrite the built-in client certificate with the new certificate to ensure that mo can connect to the server to verify that it has started correctly. This will not be necessary on evergreen, as we will use a different set of cluster configurations there.

Do I assume correctly that after Evergreen is added, we'll share the same configs for both Evergreen and Travis? IIRC, that's the case for PHPC.

Separate question: should there be a PHPLIB ticket to bump the mongo-orchestration version we use, or is that also happening in the Evergreen migration?

Reply:
> Do I assume correctly that after Evergreen is added, we'll share the same configs for both Evergreen and Travis? IIRC, that's the case for PHPC.

Yes. I plan on consolidating these scripts and fetching them from the repo directly when building on travis-ci in the future.

> Separate question: should there be a PHPLIB ticket to bump the mongo-orchestration version we use, or is that also happening in the Evergreen migration?

The last tagged release is 0.6.12, which was tagged in December 2018. I've created PHPLIB-576 to track switching to a non-tagged version that may be more up-to-date.